### DISCUSSION TITLE
Problem with icons /> becomes ↙️ and </ become ↗️

### DISCUSSION THREAD

**Discussion by Puneet56** [NvChad/NvChad#1676] [Q&A] [answered]
I have started using neovim recently and I work mainly on react. There is a problem with the / and >. When i type /> becomes ↙️ and </ become ↗️
Where can i change it. Thanks

**Comment by siduck** [comment]
could you show this in a video so it gets easy for me to look through? @Puneet56

**Reply by Puneet56** [reply]
Here is a video showing the issue.

https://user-images.githubusercontent.com/26213419/201266372-b14f8807-7698-434d-8926-d817d6eb780e.mp4

**Reply by Puneet56** [reply]
I am typing /> but it get converted into ↙️. same for </  - ↗️

**Reply by siduck** [reply]
@Puneet56 this looks more like a terminal issue so i cant help, feel free to ask in our communities ( discord, telegram, matrix ) , check readme badge for the link

**Comment by Puneet56** [comment, marked as answer]
Fixed it by disabling ligatures. Thanks
